lib/components/ArtistScreen/artist_screen_content.dart

```diff
@@ -17,66 +17,60 @@ class ArtistScreenContent extends StatefulWidget {
 }
 
 class _ArtistScreenContentState extends State<ArtistScreenContent> {
-  Future<List<BaseItemDto>?>? songs;
-  Future<List<BaseItemDto>?>? albums;
   JellyfinApiHelper jellyfinApiHelper = GetIt.instance<JellyfinApiHelper>();
 
   @override
   Widget build(BuildContext context) {
-    songs ??= jellyfinApiHelper.getItems(
-      parentItem: widget.parent,
-      filters: "Artist=${widget.parent.name}",
-      sortBy: "PlayCount",
-      includeItemTypes: "Audio",
-      isGenres: false,
-    );
+    final futures = Future.wait([
+      jellyfinApiHelper.getItems(
+        parentItem: widget.parent,
+        filters: "Artist=${widget.parent.name}",
+        sortBy: "PlayCount",
+        includeItemTypes: "Audio",
+        isGenres: false,
+      ),
+      jellyfinApiHelper.getItems(
+        parentItem: widget.parent,
+        filters: "Artist=${widget.parent.name}",
+        sortBy: "ProductionYear",
+        includeItemTypes: "MusicAlbum",
+        isGenres: false,
+      )
+    ]);
 
     return FutureBuilder(
-        future: songs,
-        builder: (context, songSnapshot) {
-          var orderedSongs = songSnapshot.data?.reversed.toList() ?? [];
+        future: futures,
+        builder: (context, snapshot) {
+          var songs = snapshot.data?.elementAt(0) ?? [];
+          var albums = snapshot.data?.elementAt(1) ?? [];
 
-          albums ??= jellyfinApiHelper.getItems(
-            parentItem: widget.parent,
-            filters: "Artist=${widget.parent.name}",
-            sortBy: "ProductionYear",
-            includeItemTypes: "MusicAlbum",
-            isGenres: false,
-          );
+          var orderedSongs = songs.reversed.toList();
```

Comment:
Try changing the `sortOrder` in the call to `jellyfinApiHelper.getItems()` instead to make it more clear what's doing on

Reply:
Oh is thede a way to have jellyfin reverse it?